include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -1250,6 +1222,60 @@ class Radiation : public ISimulationPlugin
 
 } // namespace radiation
 } // namespace plugins
+
+namespace particles
+{
+namespace traits
+{
+    template<
+        typename T_Species,
+        typename T_UnspecifiedSpecies
+    >
+    struct SpeciesEligibleForSolver<
+        T_Species,
+        plugins::radiation::Radiation< T_UnspecifiedSpecies >
+    >
+    {
+        using FrameType = typename T_Species::FrameType;
+
+        // this plugin needs at least momentumPrev1 to run
+        using RequiredIdentifiers = MakeSeq_t<
+            position<>,
+            weighting,
+            momentum,
+            momentumPrev1
+        >;
+
+        using SpeciesHasIdentifiers = typename pmacc::traits::HasIdentifiers<
+            FrameType,
+            RequiredIdentifiers
+        >::type;
+
+        // this plugin needs the radiationMask flag
+        using SpeciesHasMask = typename pmacc::traits::HasFlag<
```

Comment:
I am confused by this definition: 
The radiation mask is not necessarily required. It is an optional (old) workaround to species filters (which, due to a missing history, would cause numeric artifacts with the radiation algorithm). You added the definition `SpeciesHasMask`  here but did not include it in `line 1270` - which is fine, since it is not required. 
But why did you keep the definition here?

Reply:
I just didn't see it in one of the merge conflicts I got from rebasing. It should be fixed now.